api/src/main/java/net/kyori/adventure/sound/Sound.java

```diff
@@ -189,4 +189,13 @@
     @Override
     @NonNull Key key();
   }
+
+  /**
+   * A player of sounds.
+   *
+   * @since 4.8.0
+   */
+  interface Player {
```

Comment:
Not sure how I feel about this.

@zml2008?

Reply:
Perhaps `Performer`?

Reply:
Sound Source?

Reply:
Emitter is nice - Source clashes with `Sound.Source` which is unfortunate..

Reply:
`Emitter` works for me.

Reply:
Emitter sounds good to me too

Reply:
Emitter it is!